.github/workflows/build-actions.yml

```diff
@@ -0,0 +1,107 @@
+#  DO-NOT-REMOVE begin-copyright-block
```

Comment:
Even in the CI files we require the license statement?

Reply:
Yes